src/DataAccess/Repositories/RedisRepository.cs

```diff
@@ -67,6 +67,12 @@ public async Task<TModel> AddAsync(TModel model, Func<Guid> guidProvider)
             {
                 return null!;
             }
+
+            if (IsWritableToGraph)
```

Comment:
Are this the fixes for the errors we encouterted when trying to create OWNS relationship between robot and actionPlan?

Reply:
yes, they are